doc/en/user/source/community/gsr/dynamic_map_layer/layers-dynamicmaplayer-wholeservice.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no">
+    <title>MapImageLayer - create dynamic map layers - 4.5</title>
+
+    <link rel="stylesheet" href="https://js.arcgis.com/4.5/esri/css/main.css">
```

Comment:
This should be standalone independent of js.arcgis.com

Reply:
Thanks for your point, but I'm unable to retrieve those js and css from the resource you linked

Reply:
I only out a few moments search into it, the feedback remains that we need a way to include these examples in an open source project.

Reply:
Hmm... @jodygarnett  you do remember this is a community module right? :-D
Jokes aside, what bad can happen to js.arcgis.com that cannot also happen to esri.github.io?
They control contents in both?
Maybe what you're looking for can be found here: https://github.com/Esri/arcgis-js-api/releases/tag/4.5.0
But it's not minified, would have to be built anyways.

Reply:
Ok, looks like we need to link to the source of the javascript too: https://github.com/Esri/arcgis-js-ap
And refer to it in the documentation?

Reply:
Yeah it is a community module, so my feedback is only really for docs.
Good find, complete URL is https://github.com/Esri/arcgis-js-api

If we can mention the trademark for ESRI and ArcGIS we should be good, perhaps as a footnote? Or just some additional text.]

Here is some wording from [osgeo website](https://www.osgeo.org/projects/geonode/)

> ArcGIS Online® is a trademarks of Esri in the United States, the European Community, or certain other jurisdictions. Other companies and products mentioned may be trademarks of their respective owners.

So something similar for ArcGIS should do the trick.

Reply:
thanks @jodygarnett  and @aaime I've added notes mentioning trademarks for ESRI and ArcGIS